src/NLog/Internal/AssemblyHelpers.cs

```diff
@@ -107,5 +107,81 @@ public static Assembly LoadFromName(string assemblyName)
 #endif
         }
 
+#if !SILVERLIGHT && !NETSTANDARD1_3
+        public static string GetAssemblyFileLocation(Assembly assembly)
```

Comment:
is this just a move or are there some changes here?

Reply:
It is a move and some change in the internallogger output